src/test/fuzz/miniscript.cpp

```diff
@@ -778,6 +781,8 @@ NodeRef GenNode(F ConsumeNode, Type root_type, bool strict_valid = false) {
             auto node_info = ConsumeNode(type_needed);
             if (!node_info) return {};
             // Update predicted resource limits.
+            scriptsize += miniscript::internal::ComputeScriptLen(node_info->fragment, ""_mst, 0, node_info->k, node_info->subtypes.size(), node_info->keys.size()) + node_info->subtypes.size() - 1;
```

Comment:
You can directly pass the number of sub fragments to `ComputeScriptLen` instead of passing `0` and then adding it.
```patch
diff --git a/src/test/fuzz/miniscript.cpp b/src/test/fuzz/miniscript.cpp
index 90d752b898..490fac18ce 100644
--- a/src/test/fuzz/miniscript.cpp
+++ b/src/test/fuzz/miniscript.cpp
@@ -781,7 +781,7 @@ NodeRef GenNode(F ConsumeNode, Type root_type, bool strict_valid = false) {
             auto node_info = ConsumeNode(type_needed);
             if (!node_info) return {};
             // Update predicted resource limits.
-            scriptsize += miniscript::internal::ComputeScriptLen(node_info->fragment, ""_mst, 0, node_info->k, node_info->subtypes.size(), node_info->keys.size()) + node_info->subtypes.size() - 1;
+            scriptsize += miniscript::internal::ComputeScriptLen(node_info->fragment, ""_mst, node_info->subtypes.size(), node_info->k, node_info->subtypes.size(), node_info->keys.size()) - 1;
             if (scriptsize > MAX_STANDARD_P2WSH_SCRIPT_SIZE) return {};
             switch (node_info->fragment) {
             case Fragment::JUST_0:
```